SA/content/tool_research/flo.typ

```diff
@@ -0,0 +1,29 @@
+= flo
+
+flo is a visual and functional programming language. It features a programming
+environment that is based on blocks connected using cables, as shown in
+@flo_screenshot_1. The corresponding compiler is able to convert the visual
+arrangements and connections into Haskell code.
+
+#figure(
+  image("../../static/flo_screenshot_1.png", width: 50%),
+  caption: "Screenshot of an if function definition in flo") <flo_screenshot_1>

```

Comment:
Declare source of image (if not self-made)

Reply:
added source